### PR TITLE
Update cloud security posture README

### PR DESCRIPTION
**Pull request by eyalkraft**
## What does this PR do?

Update the integration readme to include deployment instructions for users who already have elastic-agent DaemonSet deployed and need to update it's volumes and mounts.

## Screenshots
localhost_5601_app_integrations_detail_cloud_security_posture-0 0 2_overview

### CODE REVIEW

**Review by tehilashn — APPROVED** [review]

File: packages/cloud_security_posture/docs/README.md [hunk 1]
```diff
@@ -26,3 +17,87 @@ The Kubernetes package is tested with Kubernetes 1.21.x
...
+          type: ""
+        name: varlibdockercontainers
+      - hostPath:
+          path: /var/log
+          type: ""
+        name: varlog
+      - hostPath:
+          path: /etc/kubernetes
+          type: ""
+        name: etc-kubernetes
```
Comment (oren-zohar) [inline packages/cloud_security_posture/docs/README.md:87]: nit: indentation looks off

Comment (eyalkraft) [reply packages/cloud_security_posture/docs/README.md:87]: volumes are out of the container scope this is why it looks this way.

Comment (oren-zohar) [reply packages/cloud_security_posture/docs/README.md:87]: I was talking about the indentation per volume params, there are tabs after each volume params and not in volumeMounts

Comment (eyalkraft) [reply packages/cloud_security_posture/docs/README.md:87]: not sure what you refer as volume params but the indentation is just like standard k8s yamls:
https://github.com/elastic/elastic-agent/pull/179/files#diff-1e223c6d9f97c766078b3d2642d00261ecacbdb02abf5d0f12691a4ab46e9d7eR101

**Pull request merged**